### ISSUE TITLE
Incorrect expected style types

### ISSUE THREAD

**Issue by vlinayoDxC** [chenqingspring/react-lottie#159]
The following warning message is displayed when using the **style={}** property in the Lottie component.

![image](https://github.com/chenqingspring/react-lottie/assets/110907739/e66c2c58-809f-4718-a2ee-b2d96dc68ee1)

Even thought it´s getting the correct styling at the component level, we see the warning because of the configuration used on the propTypes (or that´s what we believe is the reason)

![image](https://github.com/chenqingspring/react-lottie/assets/110907739/7705f8e0-0554-4685-8014-eae291f9f749)

Using this code logic:
![image](https://github.com/chenqingspring/react-lottie/assets/110907739/288ed49a-5d85-4456-af84-7b4e660f1e78)

So basically it does work but with a bad warning that should be fixed.
We can see the propTypes used at the moment in the file https://github.com/chenqingspring/react-lottie/blob/master/src/index.js
are expecting a string type of value in the styles but they are actually being used as an object.

![image](https://github.com/chenqingspring/react-lottie/assets/110907739/8e351562-c9a5-45db-a2a2-8b0c0574d0bc)

**Comment by vlasscontreras** [comment]
I have the same issue starting in v1.2.4. I opened #161 to fix it.
